### ISSUE TITLE
which loss function is used in xgboost?

### ISSUE THREAD

**Issue by moseshu** [dmlc/xgboost#3732]
@tqchen  
this picture is showed in Previre tab
### Hello tqcChen! which loss function is used in xgboost?there are two different  formats in this picture
![image](https://user-images.githubusercontent.com/23112888/46121626-3146f300-c247-11e8-9763-3c52ac179002.png)

**Comment by thvasilo** [comment]
The loss function is a runtime parameter, the default is linear regression. Check here: https://xgboost.readthedocs.io/en/latest/parameter.html#learning-task-parameters
